launcher/minecraft/mod/ResourcePackFolderModel.cpp

```diff
@@ -78,12 +82,28 @@ QVariant ResourcePackFolderModel::data(const QModelIndex& index, int role) const
                 default:
                     return {};
             }
+        case Qt::DecorationRole: {
+            if (column == NAME_COLUMN && (at(row)->isSymLinkUnder(instDirPath()) || at(row)->isMoreThanOneHardLink()))
+                return APPLICATION->getThemedIcon("status-yellow");
 
+            return {};
+        }
         case Qt::ToolTipRole: {
             if (column == PackFormatColumn) {
                 //: The string being explained by this is in the format: ID (Lower version - Upper version)
                 return tr("The resource pack format ID, as well as the Minecraft versions it was designed for.");
             }
+            if (column == NAME_COLUMN) {
+                if (at(row)->isSymLinkUnder(instDirPath())) {
+                    return m_resources[row]->internal_id() +
+                        tr("\nWarning: This resource is symbolicly linked from elsewhere. Editing it will also change the origonal") +
```

Comment:
.

Reply:
```suggestion
                        tr("\nWarning: This resource is symbolically linked from elsewhere. Editing it will also change the original.") +
```